src/utils/duration.js

```diff
@@ -63,16 +63,33 @@ export function millisecondsToDHMS(numericDuration) {
   return `${dhms ? '+' : ''} ${dhms}`;
 }
 
-export function getPreciseDuration(value, excludeMilliSeconds) {
+export function getPreciseDuration(value, excludeMilliSeconds, useDayFormat) {
```

Comment:
Should we modify this function to accept an `options` object instead of having 3 parameters?

```js
export function getPreciseDuration(value, { excludeMilliseconds, useDayFormat }) {
```

Reply:
Well, the value is required and the other 2 are options. That was my thinking here.